SQL/migrate-2023/V004_suit_sensors.sql

```diff
@@ -0,0 +1,6 @@
+--
+-- Implemented in PR #17607.
+-- Adds suit sensor prefs.
+--
+ALTER TABLE `ss13_characters` ADD COLUMN `sensor_setting` INT(11) DEFAULT NULL AFTER `headset_choice`;
```

Comment:
```suggestion
ALTER TABLE `ss13_characters` ADD `sensor_setting` INT(11) DEFAULT NULL AFTER `headset_choice`;
```
Per request of wezzy, modifying this